Initial pass at restructuring styling content for tailwind and Sass... (#164)
enhance-dev/enhance.dev: app/docs/md/learn/concepts/styling/index.md

title: Styling
---

Since Enhance is standards based you are free to style however you like. You can add a stylesheet to the public folder and start cowboying styles like an absolute badass or bring whatever tooling you like as long as it can output a static CSS file to the public folder of your app you are good to go.

> 

Reviewer: I'd re-order the below to be:

1. Enhance Styles
2. Plain old CSS like we used to do, but better
3. Sass
4. Tailwind

Author: People want CSS first. Then we can ease them in.

## Plain old CSS like we used to do, but better

Modern CSS is pretty damn good nowadays. Most things you used to have to drag in a build pipeline for are now part of the Web Platform. It's pretty amazing really.
Enhance outputs an HTML page so you can add `<style>` or `<link>` tags to the [`head`](/docs/learn/starter-project/head) and start writing CSS right away.

<doc-callout level="none" mark="💅🏽">

**[Read more about using plain old CSS →](/docs/learn/concepts/styling/css)**

</doc-callout>

## Tailwind

You can use whatever CSS tooling you are used to. Just configure it to output a CSS file to the public folder then reference it in your [`head`](/docs/learn/starter-project/head).

<doc-callout level="none" mark="🐈💨">

**[Read more about using Tailwind →](/docs/learn/concepts/styling/using-tailwind)**

</doc-callout>

## Sass

Again, use whatever CSS tooling you like. Just configure it to output a CSS file to the public folder then reference it in your [`head`](/docs/learn/starter-project/head).

<doc-callout level="none" mark="📠 > 📠">

**[Read more about using Sass →](/docs/learn/concepts/styling/using-sass)**

</doc-callout>

## Enhance styles

> Enhance is a "batteries included" full-stack framework so we've preconfigured a styling system that is the culmination of decades of CSS research and development into a "best of all worlds" approach to styling. It isn't what all the cool kids are using... yet, but we feel that if you give it a chance the benefits of our **deep** styling experience will become apparent.

Reviewer 2: This sentence is a bit long and complex; suggested edit to make it more concise without losing the information:

```suggestion
Enhance is a "batteries included" full-stack framework and comes with a preconfigured (and customizable) "best of all worlds" styling system, comprising decades of CSS research and development. It isn't what all the cool kids are using — yet! — but we feel that if you give it a chance, the benefits of our **deep** styling experience will become apparent.
```
